main.js

```diff
@@ -2,6 +2,9 @@ const btn = document.getElementById('btn');
 const closeButton = document.getElementById('close');
 const nav = document.querySelector('.nav');
 const list = document.querySelectorAll('.common-li');
+const seeProjectBtns = document.querySelectorAll('.btns');
```

Comment:
- [ ] Based on project requirements, you should have a JS object with all the project data that populates the popup modal:

![image](https://user-images.githubusercontent.com/77038610/151344337-75e41966-a72f-46d6-94be-809f2e0bb9ec.png)
 
Please add the JS object with project data.